README.md

```diff
@@ -1,6 +1,7 @@
 <img src="https://raw.githubusercontent.com/felangel/bloc/master/docs/assets/bloc_logo_full.png" height="60" alt="Bloc" />
 
 [![build](https://github.com/felangel/bloc/workflows/build/badge.svg)](https://github.com/felangel/bloc/actions)
+[![examples](https://github.com/felangel/bloc/workflows/examples/badge.svg)](https://github.com/felangel/bloc/actions)
```

Comment:
You can also include this badge to each package's README. :wink:

Reply:
Great idea! Done

Reply:
@netantho I mean also to packages/{bloc, flutter_bloc, ...}/README.md Next to the main build badge. :-)

![image](https://user-images.githubusercontent.com/5287596/73783702-6e54bb00-4794-11ea-9fb9-3e3664e7d258.png)